src/main/webapp/templates/eventsoy.soy

```diff
@@ -0,0 +1,7 @@
+{namespace templates.eventsoy}
+
+{template .obtainUrl}
+  {@param calendarId : string}
+  {@param timezone : string}
+  https://calendar.google.com/calendar/embed?src={$calendarId}&ctz={$timezone}
```

Comment:
Does the eventsoy.soy file integrate with the events.html file? If so how do they connect?